src/item.cpp

```diff
@@ -3765,7 +3768,7 @@ units::volume item::volume( bool integral ) const
     }
 
     if( count_by_charges() || made_of( LIQUID ) ) {
-        auto num = ret * static_cast<int64_t>( charges );
+        units::volume num = ret * static_cast<int64_t>( charges );
```

Comment:
It turns out this isn't *quite* the type that auto was inferring, it's actually this
```suggestion
        units::quantity<int64_t, units::volume_in_milliliter_tag> num = ret * static_cast<int64_t>( charges );
```
Which is super interesting in that this is the only place in the code this particular type *exists*, this particular expression actually creates this type, but the auto was hiding it.